include/seqan3/alphabet/structure/structured_aa.hpp

```diff
@@ -77,19 +77,27 @@ class structured_aa :
 
     using base_type::base_type; // Inherit non-default constructors
 
-
+#if SEQAN3_DOXYGEN_ONLY(1)0
     //!\copydoc alphabet_tuple_base::alphabet_tuple_base(component_type const alph)
-    SEQAN3_DOXYGEN_ONLY(( constexpr structured_aa(component_type const alph) {} ))
```

Comment:
We use this syntax elsewhere too right? why does it only break here?

Reply:
This change makes the complete stuff more readable.

Reply:
Well personally I disagree, I like the short syntax better as you directly spot the "this is only documentation" in every line instead of only at the top of the list. And additionally this is done in all the other places so I would refrain from changes it here because you just saw it but nowhere else. If we change this, it should be part of another PR.